docs/database/mysql.md

```diff
@@ -0,0 +1,24 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: MSSQL
+--->
+
+# Semantic Conventions for MySQL
+
+**Status**: [Experimental][DocumentStatus]
+
+The Semantic Conventions for the [MySQL](https://www.mysql.com/) extend and override the [Database Semantic Conventions](database-spans.md)
```

Comment:
```suggestion
The Semantic Conventions for the [MySQL](https://www.mysql.com/) extend the [Database Semantic Conventions](database-spans.md)
```

A nit, but I think product-specific semantic conventions shouldn't _override_ general domain-specific ones. General domain-specific semantic conventions should apply to all related products in the domain and allow consumers to interpret data without considering any product-specific details.